servers/zms/src/main/java/com/yahoo/athenz/zms/config/MemberDueDays.java

```diff
@@ -59,23 +64,23 @@ public MemberDueDays(Domain domain, Role role, Type type) {
             roleGroupDays = role.getGroupReviewDays();
         }
 
-        userDueDateMillis = ZMSUtils.configuredDueDateMillis(domainUserDays, roleUserDays);
-        serviceDueDateMillis = ZMSUtils.configuredDueDateMillis(domainServiceDays, roleServiceDays);
-        groupDueDateMillis = ZMSUtils.configuredDueDateMillis(domainGroupDays, roleGroupDays);
+        userDueDateMillis = ZMSUtils.configuredDueDateMillis(DEFAULT_USER_EXPIRY, domainUserDays, roleUserDays);
+        serviceDueDateMillis = ZMSUtils.configuredDueDateMillis(DEFAULT_SERVICE_EXPIRY, domainServiceDays, roleServiceDays);
+        groupDueDateMillis = ZMSUtils.configuredDueDateMillis(DEFAULT_GROUP_EXPIRY, domainGroupDays, roleGroupDays);
     }
 
     public MemberDueDays(Domain domain, Group group) {
 
         // for groups we only have user and service members
         // groups cannot include other groups
 
-        Integer domainUserDays = domain.getMemberExpiryDays();
-        Integer domainServiceDays = domain.getServiceExpiryDays();
+        Integer domainUserDays = null;
```

Comment:
Why did we remove the domainUserDays and domainServiceDays value?

Reply:
Sorry, this is a mistake. It has been corrected. I have restored it.